article/_posts/2021-07-29-DiSCo.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: review
+title: "The Diffusion-Simulated Connectivity (DiSCo) Challenge"
+tags: tractography tractometry diffusion
+author: "Antoine Théberge"
+pdf: "http://hardi.epfl.ch/static/events/2021_challenge/"
+---
+
+# Introduction
+The Diffusion-Simulated Connectivity (DiSCo) Challenge is a connectivity estimation challenge presented at CDMRI, a workshop organized at MICCAI 2021. The goal of the challenge is to estimate the connectivity of a synthetic phantom composed of 16 regions of interests connected by interlacing fiber bundles. The participating teams must provide their own connectivity matrix and the team with the highest correlation coefficient with the ground-truth matrix will be declared winner.
+
+# Motivations
+
+The estimation of brain structural connectivity from diffusion-weighted magnetic resonance (DW-MR) images  is still an open subject and depends on many steps. Roughly, connectivity estimation from a set of DW-MR images acquisitions goes as follow
+
+- Image denoising and resampling
+- Signal processing and modelling
+- Tractography
+- Tractogram filtering and post-processing
+- Connectivity estimation
+
+Because each step has its own set of methods and each methods has its own set of parameters, it is primordial to evaluate the performance of the pipelines currently used.
+
+# Data
+
+![](/article/images/disco/fig1.jpg)
+
+> Left: ROIs. Middle: Strands. Right: Connectivity matrix.
+
+To do so, the organizing team has released three versions (training, validation and testing) of the same dataset, each with the same regions of interests (ROIs) but different connectivity matrices. Each dataset was generated by deforming large tubular fibers to form arcs between ROIs, with energy parameters controlling the length, curvature and overlap with others. Each strand was then separated into smaller virtual axons of varying diameter.
```

Comment:
Rather than the "same dataset", if connectivity matrices are different, wouldn't they be different datasets with the same underlying model to generate the data and the same regions of interest?

Reply:
Reworded in d6aefbc0de10be4f97613718234cedae4bd20da6

Reply:
Thanks. I later realized that maybe it was indeed the same dataset, and using that data they might have explicitly disallowed some regions to be connected to generate each subset. Sorry if it was me that misunderstood the principle.